### PR TITLE
Vietnamese language related changes and German translation

### PR DESCRIPTION
**Pull request by Iix**
a few additional changes related to Vietnamese language selector

### CODE REVIEW

**Comment by Iix** [comment]
added German translation in commit 1061c34

File: locales/de-DE/App.json [hunk 1]
```diff
@@ -0,0 +1,247 @@
...
+  "(current)": "(aktuell)",
+  "Default address for sending transactions.": "Standardadresse für das Senden von Transaktionen.",
+  "Error while fetching data for address {{ hash }}": "Fehler beim Abrufen von Daten für Adresse {{ hash }}",
+  "Error while fetching pending transactions for address {{ hash }}": "Fehler beim Abrufen von ausstehenden Transaktionen für Adresse {{ hash }}",
+  "Open in explorer": "Im Explorer öffnen",
+  "Reduce": "Verkleinern:",
+  "Show addresses": "Adressen anzeigen",
+  "More info": "Mehr Infos",
+  "passphraseWarningMessage": "passphraseWarningMessage",
+  "welcomeScreenPassphraseMessage": "welcomeScreenPassphraseMessage",
```
Comment (nop33) [inline locales/de-DE/App.json:219]: These 2 are not translated properly :)

image

File: locales/de-DE/App.json [hunk 2]
```diff
@@ -0,0 +1,247 @@
...
+  "Addresses": "Adressen",
+  "Generate new address": "Neue Adresse generieren",
+  "Advanced management": "Erweiterte Verwaltung",
+  "Show operations": "Vorgänge anzeigen",
+  "Hide operations": "Vorgänge ausblenden",
+  "Consolidate (merge) your UTXOs into one.": "Ihre UTXOs in einem konsolidieren (zusammenführen).",
+  "Start": "Starten",
+  "Generate one address per group": "Eine Adresse pro Gruppe generieren",
+  "Useful for miners or DeFi use.": "Nützlich für Miner oder DeFi.",
+  "More to come...": "More to come...",
```
Comment (nop33) [inline locales/de-DE/App.json:154]: This is also not translated :)

**Review by nop33 — CHANGES_REQUESTED** [review]
Thanks a lot! I left some comments.

**Comment by Iix** [comment]
additional German translations were added, as requested

File: locales/de-DE/App.json [hunk 3]
```diff
@@ -0,0 +1,247 @@
...
+  "(current)": "(aktuell)",
+  "Default address for sending transactions.": "Standardadresse für das Senden von Transaktionen.",
+  "Error while fetching data for address {{ hash }}": "Fehler beim Abrufen von Daten für Adresse {{ hash }}",
+  "Error while fetching pending transactions for address {{ hash }}": "Fehler beim Abrufen von ausstehenden Transaktionen für Adresse {{ hash }}",
+  "Open in explorer": "Im Explorer öffnen",
+  "Reduce": "Verkleinern:",
+  "Show addresses": "Adressen anzeigen",
+  "More info": "Mehr Infos",
+  "passphraseWarningMessage": "passphraseWarnmeldung",
+  "welcomeScreenPassphraseMessage": "BegrüßungsbildschirmPassphraseMeldung",
```
Comment (nop33) [inline locales/de-DE/App.json:219]: Hey @Iix , this is not what I meant! Have a look of what the translations in the English language look like:

https://github.com/alephium/desktop-wallet/blob/master/locales/en-US/App.json#L218-L219

**Review by nop33 — CHANGES_REQUESTED** [review]

File: locales/de-DE/App.json (hunk 3 above)
Comment (Iix) [reply locales/de-DE/App.json]: 🤦 fixed now

File: locales/de-DE/App.json [hunk 4]
```diff
@@ -0,0 +1,247 @@
...
+  "Alephium doesn't have access to your wallet.\nYou are the only owner.": "Alephium hat keinen Zugriff auf Ihre Brieftasche.\nSie sind der einzige Besitzer.",
+  "Back": "Zurück",
+  "Type your recovery phrase": "Geben Sie Ihre Wiederherstellungsphrase ein",
+  "Error while importing wallet": "Fehler beim Importieren der Brieftasche",
+  "Make sure to store the words in a secure location! They are your wallet's secret recovery phrase.": "Bewahren Sie die Worte an einem sicheren Ort auf! Sie sind die geheime Wiederherstellungsphrase Ihrer Brieftasche.",
+  "All good? Let's continue!": "Alles in Ordnung? Machen wir weiter!",
+  "Everything is ready!": "Alles ist bereit!",
+  "Welcome to Alephium.": "Willkommen in Alephium.",
+  "Let's go!": "Los geht's!",
+  "welcomeScreenOneAddressPerGroupMessage": "welcomeScreenOneAddressPerGroupMessage",
```
Comment (nop33) [inline locales/de-DE/App.json:196]: Eine Sache noch! Hoffentlich die letzte :D 

Das sollte sein:

> "Advanced user: want to start with <1>one address per group for mining or DeFi?</1>"

https://github.com/alephium/desktop-wallet/blob/master/locales/en-US/App.json#L196

**Review by nop33 — CHANGES_REQUESTED** [review]

**Review by nop33 — CHANGES_REQUESTED** [review]

**Comment by nop33** [comment]
I squashed your commits and created this PR: https://github.com/alephium/desktop-wallet/pull/418

Everything is perfect now, we can close this PR :)

**Pull request closed without merging**